index.js

```diff
@@ -383,16 +377,20 @@ module.exports = {
         // It's perfectly reasonable to update/replace something
         // in the trash or unpublished, including making it live again
         // or publishing it
-        return self.find(job.req, query).trash(null).published(null).toObject(function(err, existing) {
-          if (err) {
-            return callback(err);
-          }
-          if (!existing) {
-            return callback('update-notfound ' + query[keyField]);
-          }
-          piece = existing;
-          return callback(null);
-        });
+        return self
```

Comment:
Too many style changes to locate your real changes, please resubmit without these. Thanks.

Reply:
Pushed version without style changes!